client/box/box-list.view.html

```diff
@@ -11,14 +11,53 @@
+
+      <md-button ng-if="!box.draggingEnabled &#38;&#38; !box.bulkEdit" ng-disabled="!box.data.contents.length" class="md-icon-button" aria-label="Move" ng-show="box.data.owner === main.user.name || main.user.isAdmin" ng-click="box.draggingEnabled = true">
         <md-tooltip md-direction="left">Enable dragging to move Pokémon in this box</md-tooltip>
         <i class="material-icons">open_with</i>
       </md-button>
+
+      <md-button ng-if="!box.bulkEdit &#38;&#38; !box.draggingEnabled" class="md-icon-button" aria-label="Move" ng-show="box.data.owner === main.user.name || main.user.isAdmin" ng-click="box.bulkEdit = true">
+        <md-tooltip md-direction="left">Bulk edit Pokémon</md-tooltip>
+        <i class="material-icons">edit</i>
+      </md-button>
+
+      <!-- Not implemented yet -->
+      <!-- <md-button ng-if="box.bulkEdit" class="md-icon-button" aria-label="Bulk Change Visibility" ng-show="box.data.owner === main.user.name || main.user.isAdmin" ng-click="box.bulkVisibility()">
+        <md-tooltip md-direction="left">Visibility</md-tooltip>
+        <i class="material-icons">visibility</i>
+      </md-button>
+
+      <md-button ng-if="box.bulkEdit" class="md-icon-button" aria-label="Bulk Move Box" ng-show="box.data.owner === main.user.name || main.user.isAdmin" ng-click="box.bulkMove()">
+        <md-tooltip md-direction="left">Move Box</md-tooltip>
+        <i class="material-icons">swap_horiz</i>
+      </md-button> -->
+
+      <md-button ng-if="box.bulkEdit" class="md-icon-button" aria-label="Select all" ng-click="box.selectAll()">
+        <md-tooltip md-direction="left">Select All</md-tooltip>
+        <i class="material-icons">select_all</i>
+      </md-button>
+
+      <md-button ng-if="box.bulkEdit" class="md-icon-button" aria-label="Select none" ng-click="box.selectNone()">
+        <md-tooltip md-direction="left">Select None</md-tooltip>
+        <i class="material-icons">clear</i>
+      </md-button>
+
+      <md-button ng-if="box.bulkEdit" class="md-icon-button" aria-label="Bulk Delete" ng-show="box.data.owner === main.user.name || main.user.isAdmin" ng-click="box.bulkDelete()">
+        <md-tooltip md-direction="left">Delete</md-tooltip>
+        <i class="material-icons">delete</i>
+      </md-button>
+
+      <md-button ng-if="box.bulkEdit" class="md-icon-button" aria-label="Move" ng-show="box.data.owner === main.user.name || main.user.isAdmin" ng-click="box.bulkEdit = false">
+        <md-tooltip md-direction="left">Done editing Pokémon</md-tooltip>
+        <i class="material-icons">done</i>
+      </md-button>
+
       <md-button ng-if="box.draggingEnabled" class="md-icon-button" aria-label="Done" ng-show="box.data.owner === main.user.name || main.user.isAdmin" ng-click="box.draggingEnabled = false">
         <md-tooltip md-direction="left">Done moving Pokémon</md-tooltip>
         <i class="material-icons">done</i>
```

Comment:
The checkmark icon might imply that once you click on it, pending changes will be saved. However, there's currently nothing that would be pending since a delete would happen instantly.

I'd prefer to use an "exit" icon for exiting Edit mode, but we already have an X for "select none". X everywhere else on the site means exit, so I wonder if we could find a better "select none" icon somehow 🤔 Doesn't look like https://material.io/icons/ has one, so I'm guessing that's why you picked this.

If we can't think of anything else, I'll let this go.